string caption
---
geometry_msgs/Pose[] poses
string[] labels
bool success
string message